shift_detector/precalculations/Tokenizer.py

```diff
@@ -0,0 +1,33 @@
+import re
+import pandas as pd
+
+from shift_detector.precalculations.Precalculation import Precalculation
+from shift_detector.utils.ColumnManagement import ColumnType
+
+class TokenizeIntoWords(Precalculation):
```

Comment:
The class name should also reflect that the text is converted to lowercase and should end with `~Precalculation`. Like `LowerTokenizePrecalculation`